benchmarks/IO.py

```diff
@@ -164,9 +177,10 @@ def create_parser():
     parser.add_argument(
         "-c",
         "--compressed",
-        default=False,
-        action="store_true",
-        help="Write with Snappy compression and RLE encoding",
+        default="none",
```

Comment:
Marking this here since the line above did not change. Since we are now allowing the compression types to be set instead of just turning on and off, I think it would be better for the argument to be `--compression`. We could stick with compressed and have that just trigger all commpression types (including no compression) to run. I think that would simplify things a bit. However, I do like the ability to turn things on and off as we please.

I am wondering though if we want to default to no compression or running all. I think we are at a point where we would like to see the comparsion of the compression more than we don't. Can we configure this to default to `""` and have that trigger all the compression types?

Reply:
Updated to be `--compression` and default to `""`